packages/pug/test/__snapshots__/pug.test.js.snap

```diff
@@ -1,9 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`pug .compileClient() should support module syntax in pug.compileClient(str, options) when inlineRuntimeFunctions it false 1`] = `
-"var pug = require(\\"pug-runtime\\");
+"var pug = require('pug-runtime');
```

Comment:
i have no idea if it's important for these to be double-quotes instead of single-quotes.

Reply:
This is fine :)